databricks/koalas/utils.py

```diff
@@ -495,11 +495,10 @@ def verify_temp_column_name(df: Union["ks.DataFrame", spark.DataFrame], column_n
     from databricks.koalas.frame import DataFrame
 
     if isinstance(df, DataFrame):
-        label = tuple([column_name] + ([""] * (df._internal.column_labels_level - 1)))
-        assert (
-            label not in df._internal.column_labels
+        assert all(
+            column_name != label[0] for label in df._internal.column_labels
         ), "The give column name `{}` already exists in the Koalas DataFrame: {}".format(
```

Comment:
The previous assertion was wrong. Fixed.

Reply:
Looks way better :)

Reply:
Ah, you meant the assertion, not a docstring. 
Okay I just checked it 👍